packages/cohere/src/cohere-embedding-model.ts

```diff
@@ -50,9 +45,16 @@ export class CohereEmbeddingModel implements EmbeddingModelV1<string> {
     values,
     headers,
     abortSignal,
+    providerOptions,
```

Comment:
this is an embedding model. provider options are not supported on embedding models

Reply:
resolving, as we added providerOptions to embedding models now